proto-docs/container.md

```diff
@@ -109,7 +109,7 @@ Statuses:
 | Delete | [DeleteRequest](#neo.fs.v2.container.DeleteRequest) | [DeleteResponse](#neo.fs.v2.container.DeleteResponse) |
 #### Method Get
 
-Returns container structure from `Container` smart contract storage.
+This method returns container structure from `Container` smart contract storage.
```

Comment:
Simple `Returns` is quite common, especially if you render it as a table.

Reply:
`proto-docs` will be recovered to master